Task2/task2.py

```diff
@@ -0,0 +1,154 @@
+#представим, что пользователь образованный человек и ему не придет в голову вкидывать всякую ерунду в инпут кроме [часы:минуты]
+#хотя чисто в теории, в будущем можно дописать сюда цикл, который пропускал бы только числовой нужный нам формат, но это не точно, надо подумать на досуге...
+
+import datetime
+
+hours = {1:'один' ,2:'два', 3:'три' ,4:'четыре' ,5:'пять' ,6:'шесть' ,7:'семь',
+    8:'восемь' ,9:'девять' ,10:'десять' ,11:'одинадцать' ,12:'двенадцать' ,13:'один',
+    14:'два' ,15:'три' ,16:'четыре' ,17:'пять' ,18:'шесть',
+    19:'семь' ,20:'восемь' ,21:'девять', 22:'десять', 23:'одиннадцать',
+    00:'двенадцать'}
+
+hours2 = {1:'второго' ,2:'третьего' ,3:'четвёртого' ,4:'пятого' ,5:'шестого' ,6:'седьмого',
+    7:'восьмого' ,8:'девятого' ,9:'десятого' ,10:'одиннадцатого' ,11:'двенадцатого' ,12:'первого',
+    13:'второго' ,14:'третьего' ,15:'четвёртого' ,16:'пятого' ,17:'шестого' ,18:'седьмого',
+    19:'восьмого' ,20:'девятого' ,21:'десятого' ,22:'одиннадцатого' ,23:'двенадцатого' ,00:'первого'}
+
+no_minutes = 'ровно'
+
+minutes = {1:'одна' ,2:'две', 3:'три' ,4:'четыре' ,5:'пять' ,6:'шесть' ,7:'семь',
+    8:'восемь' ,9:'девять' ,10:'десять' ,11:'одинадцать' ,12:'двенадцать' ,13:'тринадцать',
+    14:'четырнадцать' ,15:'пятнадцать' ,16:'шестнадцать' ,17:'семнадцать' ,18:'восемнадцать',
+    19:'девятнадцать' ,20:'двадцать' ,21:'двадцать одна', 22:'двадцать две', 23:'двадцать три',
+    24:'двадцать четыре' ,25:'двадцать пять' ,26:'двадцать шесть' ,27:'двадцать семь',28:'двадцать восемь',
+    29:'двадцать девять' ,31:'тридцать одна' ,32:'тридцать две' ,33:'тридцать три' ,34:'тридцать четыре',
+    35:'тридцать пять' ,36:'тридцать шесть' ,37:'тридцать семь',38:'тридцать восемь' ,39:'тридцать девять',
+    40:'сорок' ,41:'сорок одна' ,42:'сорок две' ,43:'сорок три',44:'сорок четыре'}
+
+minutes2 = {45:'пятнадцати' ,46:'четырнадцати' ,47:'тринадцати' ,48:'двенадцати' ,49:'одиннадцати',
+    50:'десяти' ,51:'девяти' ,52:'восьми' ,53:'семи' ,54:'шести' ,55:'пяти' ,56:'четырёх' ,57:'трёх',
+    58:'двух' ,59:'одной'}
```

Comment:
it's a good data structure, although there is a potential for a simplification here